modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/Edges/ObjectRelationshipEdge.tsx

```diff
@@ -15,6 +15,9 @@ const labelStyle = {
 	fill: '#FFF',
 	fontSize: '12px',
 	fontWeight: 600,
+	paddingBottom: 1,
+	paddingTop: 1,
+	"white-space": "nowrap",
```

Comment:
Hey Rebecca, I can't set the whitespace using the "whiteSpace" pattern. 
There is a type in typescript that defines the whiteSpece and I can't import it, one approach was to pass the key as a string, I can do this for all the keys so they follow the same pattern, what do you think? 

If you're thinking of something different, let me know :D

Reply:
Hey Rebeca, the SF changes didn't work very well, I changed things a little